cxx-sensors/src/main/java/org/sonar/cxx/sensors/functionsize/CxxFunctionSizeSquidSensor.java

```diff
@@ -167,9 +165,8 @@ private void publishLocInBigFunctionForProject(InputModule module, SensorContext
   private double calculatePercentual(int overThreshold, int belowThreshold){
     double total = (double)overThreshold + (double)belowThreshold;
     if (total > 0) {
-      return (overThreshold * 100.0) / ((double)overThreshold + (double)belowThreshold);
-    }
-    else {
+      return (overThreshold * 100.0) / ((double) overThreshold + (double) belowThreshold);

```

Comment:
1. function should be static
2. `return (overThreshold * 100.0) / ((double) overThreshold + (double) belowThreshold);` -> `return overThreshold / total * 100.0; // ?`